src/components/control/date.test.jsx

```diff
@@ -1,69 +1,44 @@
 import React from 'react';
-import {
-  render,
-  screen,
-} from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 import user from '@testing-library/user-event';
 import Date from './date.jsx';
 
 describe('<Date />', () => {
-  describe('onActivate API', () => {
-    it('toggles date to active when user clicks on date control', () => {
-      const onActivate = jest.fn();
+  describe('onFocus API', () => {
+    it('calls onFocus ctive when user clicks on date control', () => {
+      const onFocus = jest.fn(event => event.persist());
       render(<>
-        <Date id="test-id" onActivate={onActivate} />
+        <Date id="test-id" onFocus={onFocus} />
       </>);
 
-      user.click(screen.getByRole('textbox'));
-      expect(onActivate).toHaveBeenCalledWith(undefined);
-    });
-
-    it('toggles date and sends value at time when user clicks on date control', () => {
```

Comment:
Do we no longer have this behavior?

Reply:
Yeah. We replaced it with the native API, onFocus and OnBlur

Reply:
Yeah. We replaced it with the native API, onFocus and OnBlur